plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src/org/wso2/developerstudio/eclipse/gmf/esb/provider/FailoverEndPointItemProvider.java

```diff
@@ -61,11 +60,34 @@ public FailoverEndPointItemProvider(AdapterFactory adapterFactory) {
 		
 		super.getPropertyDescriptors(object);
 		addPropertiesPropertyDescriptor(object);
+		addBuildMessagePropertyDescriptor(object);
 	
 		return itemPropertyDescriptors;
 	}
 
 	/**
+	 * This adds a property descriptor for the Build Message feature.
+	 * <!-- begin-user-doc -->
+	 * <!-- end-user-doc -->
+	 * @generated NOT
+	 */
+	protected void addBuildMessagePropertyDescriptor(Object object) {
+		itemPropertyDescriptors.add
+			(createItemPropertyDescriptor
+				(((ComposeableAdapterFactory)adapterFactory).getRootAdapterFactory(),
+				 getResourceLocator(),
+				 getString("_UI_FailoverEndPoint_buildMessage_feature"),
+				 getString("_UI_PropertyDescriptor_description", "_UI_FailoverEndPoint_buildMessage_feature", "_UI_FailoverEndPoint_type"),
```

Comment:
Can't we create a constant for string `"_UI_FailoverEndPoint_buildMessage_feature"` and reuse in line 79 and 80?

Reply:
These are generated codes. It is the default way defined in GMF.